.gitlab-ci.yml

```diff
@@ -40,7 +40,7 @@ clang_tidy:
     - mkdir clang-tidy
 
     # Main clang-tidy run during cmake compilation
-    - CI/clang_tidy/run_clang_tidy.sh build | tee clang-tidy/clang-tidy.log
```

Comment:
maybe we should do something like 

```
min(nproc, memory / 2)
```

or so?

Reply:
hmm how can this be done in this config file?